core/tx_pool.go

```diff
@@ -270,6 +279,9 @@ func NewTxPool(config TxPoolConfig, chainconfig *params.ChainConfig, chain block
 		reorgDoneCh:     make(chan chan struct{}),
 		reorgShutdownCh: make(chan struct{}),
 		gasPrice:        new(big.Int).SetUint64(config.PriceLimit),
+
+		// Heuristic limit, reject transactions requiring over 4 slots to prevent DOS attacks
+		maxTxSize: 4 * slotSize,
```

Comment:
This is a 4x bump. I'm not sure we should bump that immediately, or do something a bit less dramatic, e.g. set it randomly so 50% retain 1x slotsize, 25% 2x and 25% 4x. Up for discussion (and please add it to PR description)

Reply:
I thought this was a solution you felt comfortable with, notice it does not enlarge resource consumption.